libs/fn/src/lib/select/option/option.component.ts

```diff
@@ -56,6 +60,14 @@ export class OptionComponent implements OnDestroy, FocusableOption {
     /** Whether this option is hidden */
     hidden = false;
 
+    /**
+     * @hidden
+     * The displayed value of the option. Used when label not passed by user. e.g. <li fn-option [value]='Apple'> Apple </li>
+     */
+    get _viewValue(): string {
```

Comment:
We can enhance this to `return this.label ?? this.elementRef...` to remove check `option.label ? option.label : option._viewValue` in `SelectComponent`.